models/Article.php

```diff
@@ -0,0 +1,90 @@
+<?php
+require_once BASE_DIR . "/services/DB.php";
+require_once BASE_DIR . "/config/constants.php";
+require_once BASE_DIR . "/services/replaceValues.php";
+
+class Article extends DB
+{
+    private $getArticleByIdSQL = "SELECT * FROM cikk WHERE id = {{id}}";
+
+    private $getArticleByUserSQL = "SELECT * FROM cikk WHERE felhasznalo = {{felhasznalo}}";
+    private $getAricleByLektorSQL =  "SELECT * FROM cikk WHERE lektor = {{lektor}}";
+    private $getAricleByLangSQL =  "SELECT * FROM cikk WHERE nyelv = {{lang}}";
+
+    private $getArticlesByKeywordsSQL = "SELECT * FROM cikk INNER JOIN cikkulcsszo ON cikk.id = cikkkulcsszo.cikk
+                                         INNER JOIN kulcsszo ON cikkkulcsszo.kulcsszo = {{keyword}}";
+    private $getArticlesByCategoriesSQL = "SELECT * FROM cikk INNER JOIN cikktemakor ON cikk.id = cikktemakor.cikk
+                                         INNER JOIN cikkkulcsszo.kulcsszo = {{categories}}";
+
+    private $insertArticleSQL = "INSERT INTO cikk (nyelv, felhasznalo, lektor, cim, status, role) VALUES " .
+"('{{lang}}', '{{felhasznalo}}', '{{lektor}}', '{{cim}}', '{{status}}', '{{role}}' )";
```

Comment:
A {{lang}} helyett legyen {{nyelv}}, mivel az adatbázisban így szerepel a nyelv id.
A fentiekre is érvényes.
Ha számot akarsz beszúrni, akkor nem kell az aposztróf

A cikknek nincs role-ja.
Helyette kimaradt a text